helm/hatchery/values.yaml

```diff
@@ -243,6 +250,21 @@ hatchery:
   # -- (bool) Whether to use internal services url. Defaults to `global.dev`.
   useInternalServicesUrl: false
 
+# -- (map) Configuration for workspace launch test crobjob
+workspaceLaunchTest:
+  # -- (bool) Whether the workspace launch test cron job is enabled.
+  enabled: true
+  # -- (string) The schedule for the workspace launch test cron job.
+  schedule: "*/20 * * * *"
```

Comment:
this is customizedable right?

Reply:
yes. We'll just have to set it in the values.yaml for any environment we're deploying it in.